### ISSUE TITLE
title() returns empty string

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#331]
Noticed in one of our basic tests a sudden error, seems > 0.17.5 (somewhere)
the `cy.title()` returning empty string. When replacing with document property started working again
```js
// used to be cy.title()
cy.document().its('title')
   .should('include', 'My App')
```

**Comment by brian-mann** [comment]
Currently `cy.title` queries for the `<title>` element but I see simply getting the `document.title` property would be a much better/cleanup/more robust solution.

**Comment by brian-mann** [comment]
Fixed in [`0.18.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0183-12182016).
